src/buildtool.unit.test.ts

```diff
@@ -10,7 +10,7 @@ describe("command generation", () => {
         };
 
         const actual = buildtool.generateCommand(options);
-        expect(actual).toBe("buildtool")
+        expect(actual).toBe("addpath('"+ process.cwd() +"/plugins');buildtool")
```

Comment:
I think in the end we'll want to use `__dirname` instead of `process.cwd()`. See above comment for why

Reply:
Yes its changed now